src/env/node.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-restricted-globals */
+import { JSDOM } from 'jsdom';
+import utils1 from 'jsdom/lib/jsdom/living/generated/utils.js';
+import utils2 from 'jsdom/lib/jsdom/utils.js';
+import { config } from '../config';
+import { setEnv } from './index';
+import { TCopyPasteData, TFabricEnv } from './types';
+
+const { implForWrapper: jsdomImplForWrapper } = utils1;
+const { Canvas: nodeCanvas } = utils2;
+
+const copyPasteData: TCopyPasteData = {};
+
+const { window: virtualWindow } = new JSDOM(
+  decodeURIComponent(
+    '%3C!DOCTYPE%20html%3E%3Chtml%3E%3Chead%3E%3C%2Fhead%3E%3Cbody%3E%3C%2Fbody%3E%3C%2Fhtml%3E'
+  ),
+  {
+    features: {
+      FetchExternalResources: ['img'],
+    },
+    resources: 'usable',
+  }
+);
+
+const fabricDocument = virtualWindow.document;
+const fabricWindow = virtualWindow;
+
+const isTouchSupported =
+  'ontouchstart' in fabricWindow ||
+  'ontouchstart' in fabricDocument ||
+  (fabricWindow &&
+    fabricWindow.navigator &&
+    fabricWindow.navigator.maxTouchPoints > 0);
+
+config.configure({
+  devicePixelRatio: fabricWindow.devicePixelRatio || 1,
+});
+
+export const getEnv = (): TFabricEnv => {
+  return {
+    document: fabricDocument,
+    window: fabricWindow,
+    isTouchSupported,
+    isLikelyNode: true,
+    nodeCanvas,
+    jsdomImplForWrapper,
+    copyPasteData,
+  };
+};
+
+setEnv(getEnv());
```

Comment:
is there a reason why node and browser are not mirrored?
this does setEnv while browser doesn't.

Reply:
Yes there is, to support SSR we can't setup env during import so it must be set lazily.
Of course there is another approach of not setting it at all and letting the dev do that.

Reply:
ok this is the issue in general with doing things at import time, it creates issues in general as a process.
Probably we should let the developer do that, but i m not ready to think about that.